### DISCUSSION TITLE
28 days restriction

### DISCUSSION THREAD

**Discussion by jpastorm** [go-co-op/gocron#258] [Q&A] [answered]
Greetings,

I understand that currently it can only be given on days 1 to 28, and we have the option to show the last day of the month.
In the case of this December, if we wanted the 29th. 

**How would that be done?**

Really thanks for the time.

**Comment by JohnRoesler** [comment, marked as answer]
Hi,

given it’s a recurring monthly event the way it works today we don’t have a way to support a day greater than 28 as not all months have those days of course. 

It’s a feature we’d like to add support for, but we’ll have to figure out the right way to do it.

In the meantime, you could use the Cron function I believe to pick that day, or you could run a Daily job and inside the job check the date and skip if it’s not the date you expect.
